genome_designer/main/xhr_handlers.py

```diff
@@ -645,6 +645,12 @@ def get_variant_list(request):
         variant_list = lookup_variant_result.result_list
         num_total_variants = lookup_variant_result.num_total_variants
 
+        # Truncate chromosme names and add a trailing '...' if longer
```

Comment:
Actually this code should go in model_views.py. Probably in this if-else structure:

https://github.com/churchlab/millstone/blob/11015679fa30cab8ec7a0154cdac3f8dec649fcc/genome_designer/main/model_views.py#L78